src/components/Navbar/styledComponents.tsx

```diff
@@ -0,0 +1,98 @@
+import styled from "styled-components";
+import { minDevice } from "../../assets/styles/breakpoints";
+import { activeLink } from "../../assets/styles/colors";
+import P from "../P";
+
+const NavbarDiv = styled.div`
+  height: 5em;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 1.16em 0 0.81em;
+
+  &.active {
+    box-shadow: 0px 1px 10px rgba(0, 0, 0, 0.08);
+  }
+
+  @media ${minDevice.desktopTablet} {
+    margin: 0 1.5em 0 1.31em;
+    height: 4em;
+    box-shadow: none;
+  }
+
+  @media ${minDevice.desktopL} {
+    margin: 0 6.25em 0 6.06em;
+    height: 4.5em;
+  }
+
+  @media ${minDevice.desktopXL} {
+    margin: 0 12.31em 0 12.5em;
+  }
+`;
+const NavbarMenu = styled.div`
+  display: flex;
+  flex-direction: column;
```

Comment:
I think @evulins extracted a Column recently, do you think this would be useful, or rather a noise here?

Reply:
Hm, I don't know. Maybe here there is no need to use this component?

Reply:
Yeah, it is fine not to, now that I think of it, I wouldn't either 👯‍♂️